### PR TITLE
Install: ensure generated kubeconfig file is not the first file in cni-conf-dir

### PR DESCRIPTION
**Pull request by tiswanso**
For now, name the generated kubeconf file "ZZZ-istio-cni-kubeconfig".

Also, fix command to find first file in cni-conf-dir.

Closes: #12

### CODE REVIEW

File: deployments/kubernetes/install/scripts/install-cni.sh [hunk 1]
```diff
@@ -134,14 +136,14 @@ fi
...
+sed -i s/__KUBECONFIG_FILENAME__/${KUBECFG_FILE_NM}/g $TMP_CONF
 
 # Use alternative command character "~", since these include a "/".
-sed -i s~__KUBECONFIG_FILEPATH__~${HOST_CNI_NET_DIR}/istio-cni-kubeconfig~g $TMP_CONF
+sed -i s~__KUBECONFIG_FILEPATH__~${HOST_CNI_NET_DIR}/${KUBECFG_FILE_NM}~g $TMP_CONF
 sed -i s~__LOG_LEVEL__~${LOG_LEVEL:-warn}~g $TMP_CONF
 
 # default to first file in `ls` output
-CNI_CONF_NAME=${CNI_CONF_NAME:-$(ls | head -n 1)}
+CNI_CONF_NAME=${CNI_CONF_NAME:-$(ls /host/etc/cni/net.d | head -n 1)}
```
Comment (venilnoronha) [inline deployments/kubernetes/install/scripts/install-cni.sh]: I think it should be `ls -1 /host/etc/cni/net.d` as in https://github.com/istio-ecosystem/cni/pull/14/files#diff-a1fbe10058a1077a440d75c2abf5f1e1R144 on PR #14. The `-1` will ensure that the file names are printed one per line.

Comment (venilnoronha) [reply deployments/kubernetes/install/scripts/install-cni.sh]: Never mind, `ls /host/etc/cni/net.d | head -n 1` will give the same output as `ls -1 /host/etc/cni/net.d | head -n 1)`.

File: deployments/kubernetes/install/scripts/install-cni.sh [hunk 2]
```diff
@@ -78,6 +78,8 @@ ${CNI_NETWORK_CONFIG}
 EOF
 fi
 
+KUBECFG_FILE_NM=${KUBECFG_FILE_NM:-ZZZ-istio-cni-kubeconfig}
```
Comment (venilnoronha) [inline deployments/kubernetes/install/scripts/install-cni.sh]: nit: I'd prefer `KUBECFG_FILE_NAME` over `KUBECFG_FILE_NM`, just to keep it consistent with the rest of the variable names.

Comment (venilnoronha) [inline deployments/kubernetes/install/scripts/install-cni.sh]: nit: Can we have a real number prefix, like `99-istio-cni-kubeconfig` or so?

Comment (tiswanso) [reply deployments/kubernetes/install/scripts/install-cni.sh]: will do

Comment (tiswanso) [reply deployments/kubernetes/install/scripts/install-cni.sh]: The numerical prefix will always be before the alphabetical prefixes and @jwendell was having problems on minikube where the file is named something that has an alphabetical prefix, e.g. something like "minikube_cni.conf".  I forget the exact name.  So, `ZZZ-` puts it at the lexicographical end for both cases.

Comment (venilnoronha) [reply deployments/kubernetes/install/scripts/install-cni.sh]: Got it, thanks!

**Pull request merged**